### PR TITLE
Add testimonials picked by the community to README

### PR DESCRIPTION
**Pull request by notevenaperson**
After the consensus in this thread: https://boards.4channel.org/g/thread/82477594#p82481070

### CODE REVIEW

**Comment by eylles** [comment]
we arent sure if the testimonials belong in the readme, there's a strawpoll that has been shared a couple times on the sneedacity general about where to put them, please vote.

https://www.strawpoll.me/45479164

File: README.md [hunk 1]
```diff
@@ -40,6 +40,23 @@ Build instructions are available [here](BUILDING.md).
...
+> — [Latter_Pin9045, redditor](https://archive.is/txQeU)
+
+> *"(took the code from sneedacity PR lol)"*
+> — [ShiningLea, Audacium developer](https://desuarchive.org/g/thread/82467576/#82475508)
+
+> *"They want to infiltrate your mind by using hate as an infiltration technique. Literally."*
+> — [cookiengineer, pinephone user](https://archive.is/eKSLy#selection-4101.0-4101.88)"
+
+> *"Replace eed with uck."*
+> — [emabrey, tenacity maintainer](https://desuarchive.org/g/thread/82428048/#82428141)
```
Comment (kostas-from-cyber-space) [inline README.md:58]: Use https://archive.is/6q7Wo instead?

**Comment by ChucksConsulting** [comment]
A poll has been set up to decide where to put the testimonials.
https://www.strawpoll.me/45479164/

**Comment by ChucksConsulting** [comment]
Please re-submit the PR to the sneedacity.org repo

**Pull request closed without merging**